tools/build/Library/ReleaseCreator.php

```diff
@@ -130,6 +130,7 @@ class ReleaseCreator
         'themes/classic/_dev',
         'themes/webpack\.config\.js$',
         'themes/package\.json$',
+        'themes/\.browserslistrc$',
```

Comment:
If this is just to exclude one file, there is the `$filesRemoveList` property above

Reply:
I doubt we will choose this solution, because the problem will remain in case you're developing using a git clone of the app

Reply:
Looking at it today